gradle/validation/owasp-dependency-check/exclusions.xml

```diff
@@ -23,43 +23,32 @@
 
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
   <suppress>
-    <notes><![CDATA[
-   file name: simple-xml-safe-2.7.1.jar
-   We use a safe fork that fixes this
-   ]]></notes>
+    <notes><![CDATA[simple-xml-safe is a safe xml-safe fork]]></notes>
     <packageUrl regex="true">^pkg:maven/com\.carrotsearch\.thirdparty/simple\-xml\-safe@.*$</packageUrl>
     <cve>CVE-2017-1000190</cve>
   </suppress>
   <suppress>
-    <notes><![CDATA[
-   file name: netty-transport-native-epoll-4.1.29.Final.jar
-   We only use netty as a client towards Zookeeper
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/io\.netty/netty\-.*@.*$</packageUrl>
-    <cpe>cpe:/a:netty:netty</cpe>
```

Comment:
FWIW this was really wrong - since this is basically saying netty isn't identified as netty - which is untrue.